src/index.js

```diff
@@ -145,41 +175,39 @@ function setBuildNumber(usePackageJsonVersion, branch, gitHubToken, gitHubRepo,
  * @return     {Promise}  A promise that resolves with the PR object.
  */
 function getPullRequest(gitHubToken, gitHubRepo, pullRequestId) {
-	const auth = `${gitHubToken}:x-oauth-basic`,
-		authToken = Buffer.from(auth).toString("base64"),
-		requestOptions = {
-			method: "GET",
-			protocol: "https:",
-			port: 443,
-			hostname: "api.github.com",
-			path: `/repos/${gitHubRepo}/pulls/${pullRequestId}`,
-			headers: {
-				Authorization: `Basic ${authToken}`,
-				"User-Agent": "TeamCity"
-			}
-		};
+	const requestOptions = {
+		method: "GET",
+		protocol: "https:",
+		port: 443,
+		hostname: "api.github.com",
+		path: `/repos/${gitHubRepo}/pulls/${pullRequestId}`,
+		headers: {
+			Token: gitHubToken,
```

Comment:
shouldn't this be:
Authorization: `Token ${githubToken}` 

based on this documentation: https://developer.github.com/v3/#authentication

Reply:
Good spot, I've fixed that. And added the accept header the docs also recommends